index.html

```diff
@@ -244,25 +234,41 @@ <h2 class="contact-me-header">
           chat please get <br />
           in touch.
         </h2>
-        <div class="input-form">
-          <input type="text" placeholder="Full name" /><input
+        <form
+          autocomplete="off"
+          action="https://formspree.io/f/xdovwjar"
+          method="post"
+          class="input-form"
+        >
+          <input
```

Comment:
Good job with using Flexbox when positioning the elements 👍  Let's follow the template and add the same amount of space between input areas and button 👍 
<img width="600" alt="Screenshot 2023-04-04 at 9 33 27 PM" src="https://user-images.githubusercontent.com/25887253/229887055-48a485b0-4001-4366-8352-fcb438d09886.png">
<img width="371" alt="Screenshot 2023-04-04 at 9 33 50 PM" src="https://user-images.githubusercontent.com/25887253/229887068-4cfe5be1-40e3-4270-bf11-91622f035aca.png">